### ISSUE TITLE
moved responses helper into hook

### ISSUE THREAD

**Issue by Kikobeats** [balderdashy/sails#2608]
Now that is possible load external load, what about moved the responses helper into a hook?

Also map more helpers. This is a [good list](https://github.com/soyjavi/zen-server/blob/master/lib/zen.constants.coffee#L155) of the most common responses for use

And finally add a blueprint response for make easy generate missing responses helper.

I can write this is the sails team agrees

**Comment by sgress454** [comment]
Hi @Kikobeats, we're all for moving our core hooks into separately-maintained modules!  This one is a bit tricky b/c the Sails generators also copy some files into `api/responses` when you do `sails new`.  We'd also want to just start with the responses we have, and add other ones as options; having an `api/responses` folder with 20 files in it would be a bit overwhelming, and a lot of the time your `409` response will be the same as your `400` (for example).

Can you say a bit more what yo mean by "add a blueprint response for make easy generate missing responses helper"?

**Comment by CWyrtzen** [comment]
Hey @Kikobeats do you have any update/inout in light of the questions asked above?

**Comment by sailsbot** [comment]
Thanks for posting, @Kikobeats.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
